rpm.fc

```diff
@@ -30,7 +30,10 @@
 /usr/share/yumex/yumex-yum-backend --	gen_context(system_u:object_r:rpm_exec_t,s0)
 /usr/share/yumex/yum_childtask\.py --	gen_context(system_u:object_r:rpm_exec_t,s0)
 
-/usr/share/rpm(/.*)?			gen_context(system_u:object_r:rpm_var_lib_t,s0)
+# These may be hardlinked, and they're not /var, so just use usr_t
+/usr/share/rpm(/.*)?			gen_context(system_u:object_r:usr_t,s0)
+# http://lists.rpm.org/pipermail/rpm-maint/2017-October/006681.html
+/usr/lib/sysimage(/.*)?			gen_context(system_u:object_r:usr_t,s0)
```

Comment:
I prefer move this definition to https://github.com/fedora-selinux/selinux-policy/blob/rawhide/policy/modules/kernel/files.fc